p2p/stream.go

```diff
@@ -36,15 +36,19 @@ func (s *Stream) Close() error {
 // Reset closes stream endpoints and deregisters it
 func (s *Stream) Reset() error {
 	s.Local.Close()
-	s.Remote.Reset()
+	s.Remote.Close()
```

Comment:
This should still be Reset, unless I'm missing something...

Reply:
I was debugging the thing and this got commited by accident